pkg/operator/controller/controller.go

```diff
@@ -39,7 +39,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
-	metrics "kubevirt.io/containerized-data-importer/pkg/monitoring/metrics/operator-controller"
+	"kubevirt.io/containerized-data-importer/pkg/monitoring/metrics/operator-controller"
```

Comment:
```suggestion
	metrics "kubevirt.io/containerized-data-importer/pkg/monitoring/metrics/operator-controller"
```

should be named as metrics, because later you use `metrics.SetupMetrics()`, no?

Reply:
but the pkg name is metrics so it's duplicate to give it an alias named metrics

Reply:
the package name should be the name of the folder by convention I think, but since this wasn't changed in this PR let's leave it as is for now

Reply:
Alex also mentioned it's not needed in Arnon's pr see: https://github.com/kubevirt/containerized-data-importer/pull/3057#discussion_r1448044345

Reply:
for me it worked manually

Reply:
It is actually needed - https://github.com/kubevirt/containerized-data-importer/pull/3057#discussion_r1448696072
> the package name should be the name of the folder by convention I think

I think we should tackle this, in this PR or later so we don't continue to have this slip into other people's PRs

Reply:
so remove the metrics alias or not? (: